### ISSUE TITLE
Videos don't play

### ISSUE THREAD

**Issue by jamesgreystone** [asciidisco/plugin.video.netflix#676]
*I'm submitting a ...*
  - [ *] bug report
  - [ ] feature request
  - [ ] support request

### Prerequisites

* [ *] Are you running the latest version?

### Description

I select a video, the loading circle comes up, then disappear, and it goes back to the listview.

[Description of the bug or feature]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expected to happen]
Video should play
**Actual behavior:** [What actually happened]
Starts to play for 2 seconds then goes back to listview of videos
### Installation

* [ *] I installed the plugin via zip from the Releases page
* [ ] I´m using the Netflix Repo
* [ ] I´m using a different source (Please tell which)

### Operating System

* [ ] Linux (x86/x64)
* [ ] OSX (x86/x64)
* [ ] Windows (x86/x64)
* [ ] Linux (ARM)
* [* ] Android

#### Additional informatin on the environment

[Descripe your environment a bit more detailed, are you using LibreElec f.e.]

Kodi 18.2
## Debug log

[Please include a link to your debug log (use something like [http://sprunge.us/](http://sprunge.us/)) or similar, please do not paste]

## Other information

[e.g. detailed explanation, related issues, suggestions how to fix, links for us to have context, etc.]

### Screenshots

[Please add a screenshot if that helps understanding your problem]

[You can erase any parts of this template not applicable to your Issue.]

**Comment by mishka81** [comment]
idem, with osmc 2019-05-19 and latest version of plugin

**Comment by jamesgreystone** [comment]
Yes it works on OSMC but I need it in another setup which is android running Kodi 18.2.

**Comment by mishka81** [comment]
switch to https://github.com/CastagnaIT/plugin.video.netflix

This one  seems to be discontinuated ...

**Comment by jakermx** [comment]
please share your log file.
